.goreleaser.yml

```
@@ -45,14 +45,14 @@ publishers:
     # Terraform CLI 0.10 - 0.11 perform discovery via HTTP headers on releases.hashicorp.com
     # For providers which have existed since those CLI versions, exclude
     # discovery by setting the protocol version headers to 5.
-    cmd: hc-releases upload-file {{ abs .ArtifactPath }} -header=x-terraform-protocol-version=5 -header=x-terraform-protocol-versions=5.0
+    cmd: hc-releases upload-file {{ abs .ArtifactPath }} -header=x-terraform-protocol-version=5 -header=x-terraform-protocol-versions=5.0 -upload-name {{ .ArtifactName }}
```

I think the `hc-releases` CLI library is picky about the equals sign here

```suggestion
    cmd: hc-releases upload-file {{ abs .ArtifactPath }} -header=x-terraform-protocol-version=5 -header=x-terraform-protocol-versions=5.0 -upload-name={{ .ArtifactName }}
```

---

aha ngl i had my doubts there b/c all the other ones use equals 😅 thanks for reviewing @bflad ! I saw yours in the other provider and figured i'd follow in suit